packages/errors/src/codes.ts

```diff
@@ -475,6 +476,7 @@ export type SolanaErrorCode =
     | typeof SOLANA_ERROR__TIMESTAMP_OUT_OF_RANGE
     | typeof SOLANA_ERROR__TRANSACTION__ADDRESS_MISSING
     | typeof SOLANA_ERROR__TRANSACTION__ADDRESSES_CANNOT_SIGN_TRANSACTION
+    | typeof SOLANA_ERROR__TRANSACTION__EMPTY__SIGNATURES_CANNOT_BE_ENCODED
```

Comment:
Jw if `__CANNOT_ENCODE_WITH_EMPTY_SIGNATURES` fits @lorisleiva's error naming pattern better here?

Reply:
Agreed! There's also double underscore after `EMPTY` currently.

Reply:
I wonder if we should just use the existing `SOLANA_ERROR__TRANSACTION__SIGNATURES_MISSING` error for this though. Because the reason you cannot encode it is simply because transactions should have at least one signature.

Reply:
I did consider it but I think it is a different error. `TRANSACTION__SIGNATURES_MISSING` is used when we try to send a transaction, or assert it's fully signed - it tells you the transaction needs these signatures. Whereas this is saying that the transaction doesn't have any expected signers. You resolve it by adding an account with a signer role, not by signing the transaction. It's fine to encode with missing signatures. We also have an `addresses: string[]` context for `TRANSACTION__SIGNATURES_MISSING` which we couldn't give a useful value to here.

Reply:
I'm happy with that. When in doubt, I think we should use separate errors to be safe.